packages/health-and-info/src/main/index.ts

```diff
@@ -0,0 +1,34 @@
+import HealthCheckMiddleware from './HealthCheckMiddleware'
+import HealthCheck from './HealthCheck'
+import EndpointComponent from './components/EndpointComponent'
+
+/**
+ * HealthCheckMiddleware class provides middleware functions for handling
+ * health and deployment information endpoints in an Express.js application.
+ *
+ * Example:
+ *
+ * ```typescript
+ * const healthMiddleware = new HealthCheckMiddleware(options);
+ * app.get('/health', healthMiddleware.health);
+ * app.get('/info', healthMiddleware.info);
+ * app.get('/ping', healthMiddleware.ping);
+ * ```
+ */
+export function healthCheckMiddleware(...args: ConstructorParameters<typeof HealthCheckMiddleware>) {
+  return new HealthCheckMiddleware(...args)
+}
+
+export function healthCheck(...args: ConstructorParameters<typeof HealthCheck>) {
```

Comment:
**question:** this doesn't need to be exposed here does it? the client won't need to call it?

Reply:
tbh I exposed this originaly so people could build their own middleware, but without exposing 'ping' and 'info', it is rather pointless.

Reply:
Removed in https://github.com/ministryofjustice/hmpps-typescript-lib/pull/7/commits/06636b1b55746c1a163e319ee3070974cb144f37